test/phoenix/live_dashboard/router_test.exs

```diff
@@ -5,7 +5,7 @@ defmodule Phoenix.LiveDashboard.RouterTest do
 
```

Comment:
We need unit tests for the MetricsLive using historical data.

Reply:
I added one simple test for it in `metrics_live_test.exs` that's pretty simplistic, but seems like a good start for a basic unit test...  I'm tempted to try and write some property tests that  build on this basic pattern to assert whatever is returned in history is present in render, but TBH, not totally sure if that will add value or not.  All the feedback aside from possibly more robust testing looks resolved to me, so I may play around with more testing, but let me know if anything else catches an eye as needing attention.  Thanks again so much for all the great feedback!